models/PackageUpdateParams.v1.yaml

```diff
@@ -0,0 +1,27 @@
+title: PackageUpdateParams.v1
+type: object
+x-examples:
+  update-picked-up:
+    picked_up: true
+  update-package-carrier:
+    carrier_name: USPS
+  update-package-recipient:
+    recipient_id: 1
+description: |
+  Update/Edit information about a Package.
+
+  [picked_up] - changes the package_state to picked up and assigns non null value to picked_up_at
+
+  [recipient_id] - update the package's intended recipient. Changes package_state to 'recipient_matched' and notifies host about their package via email. A previous recipient will stop getting notifications
+
+  [carrier_name] - change/update the package's carrier/courier information
+properties:
+  picked_up:
+    type: boolean
+    description: changes the package_state to picked up and assigns non null value to picked_up_at
+  carrier_name:
+    type: string
+    description: "change/update the package's carrier/courier information"
+  recipient_id:
+    type: integer
```

Comment:
Didn't we talk about needing/wanting to make all IDs strings?
That way we could transition to using UUIDs without an issue at some point.
@stevenallen05

Reply:
Yes, this should be a string. But it's also too late in the application development to turn that into a rule right now. Let's discuss next week about making that part of the styleguide.